scripts/in_nic/vocab/pull_data.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import json
+import pandas as pd
+
+url =  "http://vocab.nic.in/rest.php/"
+
+data_files = {
+    'States': 'states/',
+    'Country':  'country/',
```

Comment:
When you pull the country set, be aware the names are not always in English.  e.g.

{"country_id":"LY","country_name":"LIBYAN ARAB JAMAHIRIYA"}